packages/renderer/src/stores/create-network.ts

```diff
@@ -0,0 +1,57 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { ProviderContainerConnectionInfo } from '/@api/provider-info';
+
+export interface NetworkInfo {
+  networkName: string;
+  label: string;
+  interfaceName: string;
+  subnet: string;
+  ipRange: string;
+  gateway: string;
+  ipv6Enabled: boolean;
+  internalEnabled: boolean;
+  dnsEnabled: boolean;
+  dnsServers: string;
+  driver: string;
+  options: string;
+  createError: string | undefined;
+  inProgress: boolean;
```

Comment:
`createError` & `inProgress` should not be inside the NetworkInfo object, this is specific to the `CreateNetwork` form, and should be treated as individual state.